### ISSUE TITLE
Unable to compile (Identifier not found)

### ISSUE THREAD

**Issue by jose1711** [ultrastares/usdxworldparty#21]
```
..
Compiling base/UDLLManager.pas
/home/jose/tmp/ultrastardxwp-git/src/ultrastardxwp/src/base/UDLLManager.pas(169,99) Error: Identifier not found "GetLastError"
/home/jose/tmp/ultrastardxwp-git/src/ultrastardxwp/src/base/UDLLManager.pas(199,107) Error: Identifier not found "GetLastError"
/home/jose/tmp/ultrastardxwp-git/src/ultrastardxwp/src/base/UDLLManager.pas(227,30) Warning: Implicit string type conversion with potential data loss from "WideString" to "AnsiString"
/home/jose/tmp/ultrastardxwp-git/src/ultrastardxwp/src/base/UDLLManager.pas(244,33) Warning: Implicit string type conversion with potential data loss from "WideString" to "AnsiString"
/home/jose/tmp/ultrastardxwp-git/src/ultrastardxwp/src/base/UDLLManager.pas(253,31) Warning: Implicit string type conversion with potential data loss from "WideString" to "AnsiString"
/home/jose/tmp/ultrastardxwp-git/src/ultrastardxwp/src/base/UDLLManager.pas(261,28) Warning: Implicit string type conversion with potential data loss from "WideString" to "AnsiString"
UDLLManager.pas(267) Fatal: There were 2 errors compiling module, stopping
Fatal: Compilation aborted
Error: /usr/bin/ppcx64 returned an error exitcode
make[1]: *** [Makefile:256: ../game/WorldParty] Error 1
```

**Comment by jose1711** [comment]
seems to be caused by 19b527b6f8a09d3ad188e7ea4d1c98a1d4022688 which introduced this win32 platform specific call

**Comment by TeLiXj** [comment]
thanks for report this error, I will try to resolve it as soon as posible

**Comment by skerit** [comment]
@TeLiXj Did you perhaps get a chance to look at the issue? :)

**Comment by TeLiXj** [comment]
> @TeLiXj Did you perhaps get a chance to look at the issue? :)

This week will be resolved ;)

**Comment by TeLiXj** [comment]
Fixed temporaly with last commit on alfa fb98123

**Comment by jose1711** [comment]
thank you, alfa branch works for me
